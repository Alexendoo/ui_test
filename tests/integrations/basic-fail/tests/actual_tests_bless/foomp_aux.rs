//@aux-build:foomp.rs

use foomp::add;

fn main() {
    add("42", 3);
    //~^ ERROR: mismatched types
}
