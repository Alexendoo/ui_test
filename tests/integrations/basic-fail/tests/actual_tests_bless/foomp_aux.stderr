error[E0308]: mismatched types
 --> $DIR/foomp_aux.rs:6:9
  |
6 |     add("42", 3);
  |     --- ^^^^ expected `u8`, found `&str`
  |     |
  |     arguments to this function are incorrect
  |
note: function defined here
 --> $DIR/tests/integrations/basic-fail/$DIR/auxiliary/foomp.rs:1:8
  |
1 | pub fn add(_: u8, _: u8) {}
  |        ^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
